plugins/builtin/strategy/target-value/plugin/plugin.go

```diff
@@ -131,23 +172,22 @@ func (s *StrategyPlugin) Run(req strategy.RunRequest) (strategy.RunResponse, err
 // occur, if any at all. It takes into account the current task group count in
 // order to correctly account for 0 counts.
 //
-// TODO(jrasell) the direction should probably be a type, rather than a plain
-// 	string so we don't have to return an empty string for no direction.
-func (s *StrategyPlugin) calculateDirection(count int64, factor float64) string {
+// The input factor value is padded by e, such that no action will be taken if
+// factor is within [1-e; 1+e].
+func (s *StrategyPlugin) calculateDirection(count int64, factor, e float64) scaleDirection {
 	switch count {
 	case 0:
 		if factor > 0 {
-			return "up"
-		} else {
-			return "down"
+			return scaleDirectionUp
 		}
+		return scaleDirectionDown
```

Comment:
why is this `scaleDirectionDown` and not `scaleDirectionNone`? if it is unreachable code, we should add a comment (and use an appropriate "zero" value)

Reply:
Yeah, it seems like this should return `scaleDirectionNone`. It doesn't make sense to scale down if `count` is `0`.

@jrasell any thoughts?